rslib/src/deckconfig/schema11.rs

```diff
@@ -86,7 +86,7 @@ pub struct DeckConfSchema11 {
     #[serde(default = "wait_for_audio_default")]
     wait_for_audio: bool,
     #[serde(default)]
-    sm2_retention: f32,
+    historical_retention: f32,
```

Comment:
You might want to keep it as sm2_retention in this file, and just add `/// doccomment` that it is historical_retention. Otherwise, users who have downgraded their proflie, and then upgrade with a new Anki version, will lose their existing settings.